packages/react-server/src/ReactFizzServer.js

```diff
@@ -882,6 +883,41 @@ function createClassComponentStack(
     type,
   };
 }
+function createServerComponentStack(
+  task: Task,
+  debugInfo: void | null | ReactDebugInfo,
+): null | ComponentStackNode {
+  // Build a Server Component parent stack from the debugInfo.
+  if (__DEV__) {
+    let node = task.componentStack;
+    if (debugInfo != null) {
+      const stack: ReactDebugInfo = debugInfo;
+      for (let i = 0; i < stack.length; i++) {
+        const componentInfo: ReactComponentInfo = (stack[i]: any);
+        if (typeof componentInfo.name !== 'string') {
+          continue;
+        }
+        let name = componentInfo.name;
+        const env = componentInfo.env;
+        if (env) {
+          name += ' (' + env + ')';
+        }
+        node = {
+          tag: 3,
+          parent: node,
+          type: name,
+          owner: componentInfo.owner,
+          stack: componentInfo.stack,
+        };
+      }
+    }
+    return node;
+  }
+  // eslint-disable-next-line react-internal/prod-error-codes
```

Comment:
Is the reason here that this error shouldn't appear in prod bundles? We usually call this out explicitly e.g.  https://github.com/facebook/react/blob/d40ea875a42d5726e8a7e5a6f47db2dbe314957d/packages/react-server/src/ReactFlightServer.js#L3269-L3273

Reply:
> say anything more than what the disable line says anyway so it's a classic unnecessary commment.

To me the eslint-disable doesn't say anything. I think it's only useless if you have high context.